server/Context/SampleData.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.EntityFrameworkCore;
+using server.Models.Domain;
+
+namespace server.Context
+{
+    public static class SampleData
+    {
+        public static void CreateSampleData(ModelBuilder modelBuilder)
+        {
+            // Sample users
+            modelBuilder.Entity<User>().HasData(new User(1, "Code Master Flex", "code_master@example.com"));
+            modelBuilder.Entity<User>().HasData(new User(2, "Debug Diva", "debug_diva@example.com"));
+            // Sample rooms
+            modelBuilder.Entity<Room>().HasData(new Room(1, "Binary Bunker"));
+            modelBuilder.Entity<Room>().HasData(new Room(2, "Algorithm Alcove"));
+            // Sample seats
+            modelBuilder.Entity<Seat>().HasData(new Seat(1, 1));
```

Comment:
Kan vi legge til litt flere seats i hvert rom?
Store rommet: 10 plasser
Lille rommet: 5 plasser

Reply:
Det kan jeg fikse. Legger til 10 seats og 5 seats for store og lille rommet.